### PR TITLE
Add Baguette theme to DenizenMetaDoc

### PR DESCRIPTION
**Pull request by apademide**
V1.0.0 of Apademide's Baguette Theme

### CODE REVIEW

File: Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -51,6 +51,7 @@
                         <a class="dropdown-item" title="created by Chris|LordNoob" onclick="changetheme('chrispy')">Chrispy Dark</a>
                         <a class="dropdown-item" title="created by Breadcrumb" onclick="changetheme('breadcrumb')">Breadcrumb Dark</a>
                         <a class="dropdown-item" title="created by Alex 'mcmonkey' Goodwin" onclick="changetheme('light')">Quite Light</a>
+		        <a class="dropdown-item" title="created by Apademide" onclick="changetheme('baguette')">Quite Light</a>
```
Comment (mcmonkey4eva) [inline Views/Shared/_Layout.cshtml]: spacing went wonky, and the name is wrong

File: wwwroot/css/theme/colors_apademide.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
+/*                                                                                          */
+/*                             Stylesheet made with love by                                 */
+/*                                                                                          */
+/*              A     PPPP      A      DDDD    EEEEE  M     M  II  DDDD    EEEEE            */
+/*             AAA    PP  PP   AAA     DD  DD  EE     MM   MM  II  DD  DD  EE               */
+/*            AA AA   PP  PP  AA AA    DD  DD  EEEE   MMM MMM  II  DD  DD  EEEE             */
+/*           AAAAAAA  PPPP   AAAAAAA   DD  DD  EE     MM M MM  II  DD  DD  EE               */
+/*          AAA   AAA PP    AAA   AAA  DDDD    EEEEE  MM   MM  II  DDDD    EEEEE            */
```
Comment (mcmonkey4eva) [inline wwwroot/css/theme/colors_apademide.css]: no

File: wwwroot/css/theme/colors_apademide.css [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+.script_normal{color:#FFFFFF;}
+.script_def_name{color:#fee75c;text-transform:uppercase;}
+/* Denizen Command Syntax Colors */
+.syntax_command{color:#ff8bf3;font-style:italic;}
+.syntax_required{color:#696cfc;font-weight:bold;}
+.syntax_optional{color:#8aecf3;}
+.syntax_default{color:#7fff7f;font-style:italic;}
+.syntax_fillable{color:#a1a1ac;}
+.syntax_list{color:#EEEEFF;}
+.syntax_colon{color:#00bacc;font-weight:bold;}
```
Comment (mcmonkey4eva) [inline wwwroot/css/theme/colors_apademide.css]: NL@EoF

File: wwwroot/css/theme/colors_apademide.css [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+/* NavBar */
+.navbar{font-size: 16px;box-shadow:0px 1px 10px var(--shadow-color);padding: 10px 20px;}
+nav.bg-primary{background-color:#fff48f!important;}
+.navbar-dark .navbar-brand{position:relative;font-weight:bold;text-decoration:none;text-shadow:1px 0px #222,2px 2px 0px #222,3px 3px 0px #222,-1px 1px 0px #222,0px 2px 0px #222,1px 3px 0px #222,-1px -1px 0px #222,-1px 1px 0px #222,1px -1px 0px #222;}
+.navbar-dark .navbar-nav .nav-link,.navbar-dark .navbar-nav .nav-link:hover{color:var(--very-black);text-decoration:none;}
+.navbar-dark .navbar-nav .nav-item{position: relative;}
+.navbar-dark .navbar-nav .nav-item:after,.navbar-brand:after{content:'';background-color:white;position:absolute;height:3px;bottom:5px;right:100%;left:0;transition:all .3s cubic-bezier(0.85,0,0.15,1);}
+.navbar-dark .navbar-nav .nav-item:hover:after{right:7px;left:8px;box-shadow:1px 1px 0px 2px #222;}
+.navbar-brand:hover:after{right:0px;left:0px;box-shadow:1px 1px 0px 2px var(--shadow-color);}
+/* Activate that set of rules for more *Frenchiness* */
```
Comment (mcmonkey4eva) [inline wwwroot/css/theme/colors_apademide.css]: What.

File: ThemeHelper.cs [hunk 5]
```diff
@@ -62,6 +62,10 @@ static ThemeHelper()
             breadcrumb.ColorCSS = "/css/theme/colors_breadcrumb.css";
             breadcrumb.Footer = "Theme: Breadcrumb Dark, created by Breadcrumb";
             Themes.Add("breadcrumb", breadcrumb);
+            ThemeHelper baguette = Dark.MemberwiseClone() as ThemeHelper;
```
Comment (mcmonkey4eva) [inline ThemeHelper.cs]: This is a light theme apparently, so work off the light base not the dark

Comment (apademide) [reply ThemeHelper.cs]: should use `lightRef.MemberwiseClone()` instead or `Dark`?

Comment (mcmonkey4eva) [reply ThemeHelper.cs]: yes

**Pull request merged**